examples/models/__init__.py

```diff
@@ -16,6 +17,7 @@
     "emformer_predict": ("emformer_rnnt", "EmformerRnntPredictorModel"),
     "emformer_join": ("emformer_rnnt", "EmformerRnntJoinerModel"),
     "llama2": ("llama2", "Llama2Model"),
+    "lstm": ("lstm", "LSTMModel"),
```

Comment:
This is a common across all backends, I guess each backend specifies a whitelist of supported models, if so, this is OK to me.